dist/node/admin.js

```diff
@@ -793,11 +796,11 @@ var StitchAdminClient = exports.StitchAdminClient = function (_StitchClient) {
               };
             },
 
-            realm: function realm() {
-              var realmUrl = appUrl + '/realm';
+            sync: function sync() {
+              var syncUrl = appUrl + '/sync';
```

Comment:
is this intentional?

Reply:
Yeah this got changed [a little while ago](https://github.com/mongodb/stitch-js-sdk/commit/acde8cdae6d6dcde8a8c76cbef142580a20827de#diff-75d19e3da4632ee1f6cc2cfd233abef0)